src/stories/stepper/index.tsx

```diff
@@ -1,6 +1,11 @@
-import { Stepper as ZendeskStepper } from "@zendeskgarden/react-accordions";
+import {
+  Accordion,
```

Comment:
in questo caso conviene prendere l'accordion dalla libreria, in modo da ereditarne future personalizzazioni